packages/medusa-cli/src/commands/login.js

```diff
@@ -29,52 +32,58 @@ module.exports = {
       },
     ]
 
+    console.log()
+    console.log("Login to Medusa Cloud")
+    console.log()
+
     await inquirer.prompt(qs).then(async a => {
       if (a.open === "n") {
         process.exit(0)
       }
 
-      const bo = await open(`${loginHost}${urls.browser_url}`, {
+      const bo = await open(loginUri, {
         app: "browser",
         wait: false,
       })
       bo.on("error", err => {
         console.warn(err)
-        console.log(
-          `Could not open browser go to: ${loginHost}${urls.browser_url}`
-        )
+        console.log(`Could not open browser go to: ${loginUri}`)
       })
+    })
 
-      const fetchAuth = async (retries = 3) => {
-        try {
-          const { data: auth } = await axios.get(`${authHost}${urls.cli_url}`, {
-            headers: { authorization: `Bearer ${urls.cli_token}` },
-          })
-          return auth
-        } catch (err) {
-          if (retries > 0 && err.http && err.http.statusCode > 500)
-            return fetchAuth(retries - 1)
-          throw err
-        }
+    const spinner = logger.activity(`Waiting for login at ${loginUri}`)
+    const fetchAuth = async (retries = 3) => {
+      try {
+        const { data: auth } = await axios.get(`${authHost}${urls.cli_url}`, {
+          headers: { authorization: `Bearer ${urls.cli_token}` },
+        })
+        return auth
+      } catch (err) {
+        if (retries > 0 && err.http && err.http.statusCode > 500)
```

Comment:
Should we not retry on status code 500? I'm thinking `err.http.statusCode >= 500` is necessary?

Reply:
In this case it doesn't make sense; we only want to retry if the server goes down or is not responding - 500 error codes indicate that there is something wrong with the logic of the application due to an error not being handled correctly.